packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -582,24 +578,21 @@ describe('Grant Resolvers', (): void => {
           }
         })
 
-      expect(response.success).toBe(false)
-      expect(response.code).toBe('401')
-      expect(response.message).toBe('Grant Id is not provided')
+      expect(gqlQuery).rejects.toThrow(ApolloError)
+      expect(gqlQuery).rejects.toThrow('Grant id is not provided')
```

Comment:
should we check that the correct `error.extensions.code` was thrown as well?

Reply:
Tbh, I also skipped that because I didn't know how to test test.